src/content/get-started/install/openshift.mdx

```diff
@@ -0,0 +1,192 @@
+---
+
+import variables from '../../variables.json';
+import CodeBlock from '@theme/CodeBlock';
+
+This guide will walk you through the process of installing Okteto in Red Hat OpenShift.
+
+## Installation Requirements
+
+Before you start, make sure you have the following CLIs installed in your machine:
+
+- `kubectl` >= 1.28 ([kubectl installation guides](https://kubernetes.io/docs/tasks/tools/#kubectl))
+- `helm` >= 3.14 ([helm installation guides](https://helm.sh/docs/intro/install/))
+- `oc` >= 4.0.0 
+
+You'll also need the following:
+
+- An Okteto License
+- A Kubernetes cluster with OpenShift (≥ 4.0.0) installed
+
+For this guide, you will need to obtain the following values:
+
+- `<your-openshift-subdomain>`: for example, `okteto.r9yz.p2.openshiftapps.com`
+- `<cluster-name>`: for example, `okteto`
+
+## Getting your Okteto License
+
+A license is mandatory to use Okteto. You'll receive a license key as part of your subscription to Okteto. If you haven't received it, [please open a support ticket](https://okteto.com/support).
+
+If you are interested in evaluating Okteto, [sign up for our free 30 days trial](https://www.okteto.com/free-trial/). No credit card required.
+
+## Steps before Installing Okteto
+
+Before installing Okteto, you will need to follow these steps:
+
+- Create the `okteto` namespace
+- Allow containers as `root` in the `okteto` namespace
+- Allow Okteto Build service privileged access
+- Allow Okteto Daemon service to access the host
+- Copy Openshift wildcard certificate to the `okteto` namespace
+- Get the OpenShift Ingress Controller IP
+
+### Create the `okteto` namespace
+
+Run the following command to create the `okteto` namespace:
+
+```bash
+kubectl create namespace okteto
+```
+
+### Allow containers as `root` in the `okteto` namespace
+
+By default, running `root` containers in OpenShift is not allowed. Today, most of the Okteto Control Plane runs as `root` (this is something we will fix soon). To allow `root` containers for the Okteto Control Plane components, run the following commands:
+
+```bash
+oc adm policy add-scc-to-user anyuid -z default -n okteto 
+oc adm policy add-scc-to-user anyuid -z okteto -n okteto 
+oc adm policy add-scc-to-user anyuid -z okteto-buildkit -n okteto 
+oc adm policy add-scc-to-user anyuid -z okteto-registry -n okteto 
+```
+
+### Allow Okteto Build service privileged access
+
+By default, running privileged containers in OpenShift is not allowed. This restriction impacts the installation of the Okteto Build. To give privileged access to the Okteto Build service, run the following command:
+
+```bash
+ oc adm policy add-scc-to-user privileged -z okteto-buildkit -n okteto
+```
+
+### Allow the Okteto Daemon service to access the host
+
+By default, access to the host in OpenShift is not allowed. This restriction impacts the installation of the Okteto Daemon service. To give access to the host to the Okteto Daemon run the following command:
```

Comment:
This is a big permission ask. Can we be explicit on why this is needed, the impact, etc? Take into consideration that this might require the installer to go an ask for exceptions, so the more information we can give them the better.